packages/dashboard/src/components/resizablePanes/index.tsx

```diff
@@ -58,6 +60,11 @@ export const ResizablePanes: FC<ResizablePanesProps> = ({ leftPane, centerPane,
   const [isRightPaneCollapsed, setRightPaneCollapsed] = useState(selectedWidget?.length === 0);
   const [isLeftPaneCollapsed, setLeftPaneCollapsed] = useState(selectedWidget?.length === 0);
 
+  const resourceExplorerPaneWidth = Math.min(
+    Math.max(leftPaneWidth, MINIMUM_LEFT_SIDE_PANE_WIDTH),
```

Comment:
can we have these checks in the block where leftPaneWidth gets updated ? instead of again calculating after the fact ?

Reply:
resolved